controllers/admin/AdminPsThemeCustoConfiguration.php

```diff
@@ -502,19 +502,19 @@ public function setFinalList($aList)
                 if ($sType == 'pages') {
                     foreach ($aElementsList as $sController => $aPage) {
                         $aModuleFinalList[$sSegmentName][$sType][$sController]['name'] = $sController;
-                        $aModuleFinalList[$sSegmentName][$sType][$sController]['displayName'] = $this->l($aPage[0]);
+                        $aModuleFinalList[$sSegmentName][$sType][$sController]['displayName'] = $this->trans($aPage[0]);
```

Comment:
Now, trans are not escaped by default. Is there a risk that $aPage[0] contains user entry ? If yes, I think you can use htmlspechialchars on the parameter.

Reply:
I'm not sure tbh I'm investing

Reply:
To me it's not user's input